### PR TITLE
Ec 270

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: src/main/scala/org/encryfoundation/explorer/ExplorerApp.scala [hunk 1]
```diff
@@ -46,6 +48,18 @@ object ExplorerApp extends App {
     TransactionsApiRoute(TransactionService[IO](transactor, ec), settings.restApi)
   )
 
-  val combinedRoute: Route = apiRoutes.map(_.route).reduce(_ ~ _)
+  val withLogger: Directive0 = {
```
Comment (Bromel777) [inline src/main/scala/org/encryfoundation/explorer/ExplorerApp.scala]: remove braces

File: src/main/scala/org/encryfoundation/explorer/utils/Logging.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.encryfoundation.explorer.utils
+
+import com.typesafe.scalalogging.{Logger, StrictLogging}
+
+trait Logging extends StrictLogging {
+  implicit val log: Logger = logger
+
+  def logInfo(message: String): Unit = log.info(message)
```
Comment (kapinuss) [inline src/main/scala/org/encryfoundation/explorer/utils/Logging.scala]: these funs are useless

File: src/main/scala/org/encryfoundation/explorer/ExplorerApp.scala [hunk 3]
```diff
@@ -3,18 +3,20 @@ package org.encryfoundation.explorer
...
 import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Directive0, ExceptionHandler, Route}
 import akka.stream.ActorMaterializer
 import cats.effect.IO
 import doobie.hikari.HikariTransactor
 import org.encryfoundation.explorer.db.services.{HistoryService, TransactionService}
 import org.encryfoundation.explorer.http.api.routes.{ApiRoute, HistoryRoute, TransactionsApiRoute}
 import org.encryfoundation.explorer.settings.ExplorerAppSettings
+import org.encryfoundation.explorer.utils.Logging
+
```
Comment (kapinuss) [inline src/main/scala/org/encryfoundation/explorer/ExplorerApp.scala]: please dont add empty lines between imports

File: src/main/resources/application.conf [hunk 4]
```diff
@@ -2,7 +2,6 @@
   encry {
     # App data directory
     directory = ${user.dir}"/encry/data"
-
```
Comment (kapinuss) [inline src/main/resources/application.conf]: please dont touch app.conf without real need

**Pull request merged**